Source/Common/EnumHelper.cs

```diff
@@ -1,13 +1,19 @@
 namespace Schema.NET
 {
     using System;
+    using System.Diagnostics;
     using System.Diagnostics.CodeAnalysis;
 
     /// <summary>
     /// Helper for parsing strings into Enum values.
     /// </summary>
     internal static class EnumHelper
     {
+        private const string HttpSchemaOrgUrl = "http://schema.org/";
+        private const int HttpSchemaOrgLength = 18; // equivalent to "http://schema.org/".Length
+        private const string HttpsSchemaOrgUrl = "https://schema.org/";
```

Comment:
Here's that constant we were looking for.

Reply:
Ahhhhh, yeah. Should we move this out somewhere else so it doesn't sit in the `EnumHelper` if we are referencing it in the `JsonLdContext` class?